openassessment/assessment/errors/staff.py

```diff
@@ -0,0 +1,33 @@
+"""
+Errors for the staff assessment api.
+"""
+
+
+class StaffAssessmentError(Exception):
```

Comment:
As with the request error class, I wonder if it would be better to have a generic ``AssessmentError`` class that this subclasses. In most situations, the client wants to catch all assessment errors, I would think.